README.md

```diff
@@ -1,2 +1,97 @@
-# hello-world
-hello world project with microverse
+# HTML & CSS3 Course
```

Comment:
- [ ] Kindly use [this template](https://github.com/microverseinc/readme-template) for readme.